04.class/operateInterface.js

```diff
@@ -0,0 +1,97 @@
+import readline from "readline";
+import enquirer from "enquirer";
+
+const { Select } = enquirer;
+export default class OperateInterface {
+  constructor(memoOperation) {
+    this.memoOperation = memoOperation;
+  }
+
+  async listMemos() {
+    try {
+      const rows = await this.memoOperation.fetchMemos();
+      rows.forEach((row) => {
+        console.log(row.content.split("\n")[0]);
+      });
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async readMemos() {
+    try {
+      const result = await this.#selectMemosFromList("see");
+      console.log(result);
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async deleteMemo() {
+    try {
+      const result = await this.#selectMemosFromList("delete");
+      this.memoOperation.removeMemo(result);
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async writeMemo() {
+    try {
+      const content = await this.#createMemoPrompt();
+      await this.memoOperation.addMemo(content);
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async #selectMemosFromList(action) {
+    try {
+      const rows = await this.memoOperation.fetchMemos();
```

Comment:
```suggestion
      const memos = await this.memoOperation.fetchMemos();
```

こっちをmemosと呼んだ方が自然な気がします

Reply:
rowsやresultの部分の扱いについて少し不安がありましたが、今回のご指摘で明確にできました。
ありがとうございます！
修正します！